exec/java-exec/src/test/java/org/apache/drill/test/OperatorFixture.java

```diff
@@ -86,37 +117,35 @@
  * <li>Multiple threads of execution.</li>
  * </ul>
  */
-
 public class OperatorFixture extends BaseFixture implements AutoCloseable {
 
+  public OperatorContext operatorContext(PhysicalOperator config) {
+    return new MockOperatorContext(context, allocator(), config);
+  }
+
   /**
    * Builds an operator fixture based on a set of config options and system/session
    * options.
    */
-
-  public static class OperatorFixtureBuilder
+  public static class Builder
```

Comment:
Your PR is accidentally undoing a name change from a prior commit. Please check if we have any other collisions in this or related files.

Reply:
There are no collisions. Java has a nice syntax for handling names like this. Specifically you can use this name for the class **OperatorFixture.Builder** in all your variable declarations. This removes the redundancy of prefixing the name of an inner class with the name of the outer class.

Reply:
Yes, of course. But one of the changes was to pull this out to a new top-level class. In that case, "Builder" by itself is rather generic.

Reply:
It's a nested class in the master branch currently https://github.com/apache/drill/blob/master/exec/java-exec/src/test/java/org/apache/drill/test/OperatorFixture.java . So I didn't change that in this PR. Did you want me to pull it out into a top level class as part of this PR?